README.md

```diff
@@ -36,6 +36,7 @@ The various digital twins serve different purposes, including analysis, control,
 ## Videos
 
 - [Talks of the Engineering Digital Twins – Seminar Series](https://edt.community/past-events/)
+- [Computerphile video on Digital Twins](https://www.youtube.com/watch?v=2ryz9IPIQes)
```

Comment:
It would be nice if there was a short note why this video is awesome. (I know, because the speaker is awesome, but is there another thing to note? Great general overview?)

Reply:
Added :-)